documentation/interpretation_guide.md

```diff
@@ -73,24 +83,32 @@ If there are restrictions for the circulation of private cars based on certain c
 
 ### C8-  International Travel Controls
 
+- This indicator does not have a binary flag variable to reflect geographic variation in policies. This implies that this policy applies generally across the whole jurisdiction. Therefore, we generally code the level of policy that applies everywhere across the jurisdiction- the highest common value of policy nationwide or statewide. We do not code any policies of a higher value that are only available in limited areas.  *For example* if one airport is screening passenger arrivals, but the majority of them are not, this would not be coded as a 1. All airports across the country would need to be screening before this policy could be seen to apply generally and be coded.
 - If visitors are meant to self quarantine/isolate after travelling to certain areas, but there are no enforcement or tracking measures in place, we record the official policy accordingly, and make a note that the situation on the ground may not reflect this.
 - If a country is still repatriating citizens from overseas, but otherwise has closed borders, this still counts as a full border closure (4). This indicator focuses largely on requirements for foriegn passengers, in the sense that even during “total closure”, countries will still let their citizens cross the border to return home (although may subject them to quarantine).
 - **For subnational (state-level) data:** As international travel is usually an issue for the federal government, subnational governments will rarely implement relevant policies for C8. The sorts of state-level activity that would be recorded under C8 include when individual airports proactively screen passengers, or if states created their own mandatory quarantines for international travelers.
+- If a country’s borders are completely closed, but due to a civil war, or other non-covid related reasons, this should still be coded as a 4. While this is not a covid specific policy, in practice the border is still closed so we want to best represent the lived reality of people living there, in order to understand the effect this could have on COVID-19.
```

Comment:
This is written for a coder ("this should still be coded...") rather than a user ("this will still be reported...")